.reuse/dep5

```diff
@@ -0,0 +1,6 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Source: https://github.com/RevolutionPi/revpi-hat-eeprom
+
+Files: Cargo.lock
+Copyright: KUNBUS GmbH
```

Comment:
I'm surprised there is not copyright year here, but checking REUSE compliance with the tool seems to be okay. I guess the year is not required?

Reply:
I dropped the year, as the file is generated and updating the year is a useless chore ;)